### ISSUE TITLE
gradle中怎么不能呢compile出来了? 是还地址了吗?

### ISSUE THREAD

**Issue by wushge11** [w446108264/XhsEmoticonsKeyboard#19]
compile 'com.github.w446108264:XhsEmoticonsKeyboard:2.0.3'

**Comment by w446108264** [comment]
allprojects {
    repositories {
        jcenter()
        maven { url "https://jitpack.io" }
    }
}

jitpack加了吗

**Comment by w446108264** [comment]
compile 'com.github.w446108264:XhsEmoticonsKeyboard:2.0.2'

**Comment by wushge11** [comment]
嗯嗯  我 马虎了 谢谢你



wushange@126.com
 
发件人： zhongdaxia
发送时间： 2016-05-23 15:52
收件人： w446108264/XhsEmoticonsKeyboard
抄送： wushge11; Author
主题： Re: [w446108264/XhsEmoticonsKeyboard] gradle中怎么不能呢compile出来了? 是还地址了吗? (#19)
allprojects {
repositories {
jcenter()
maven { url "https://jitpack.io" }
}
}
jitpack加了吗

**Comment by kadam5mohit** [comment]
Please tell me latest version of this library . I have integrate compile 'com.github.w446108264:XhsEmoticonsKeyboard:2.0.2' gradel . 

Please tell me how can i  implement this keyboard in my app .

**Comment by kadam5mohit** [comment]
When i am going to write below code as given instruction then compilation error coming :
 EmoticonPageSetEntity xhsPageSetEntity
                = new EmoticonPageSetEntity.Builder()
                .setLine(3)
                .setRow(7)
                .setEmoticonList(ParseDataUtils.ParseXhsData(DefXhsEmoticons.xhsEmoticonArray, ImageBase.Scheme.ASSETS))
                .setIPageViewInstantiateItem(SimpleCommonUtils.getDefaultEmoticonPageViewInstantiateItem
                         (getCommonEmoticonDisplayListener(emoticonClickListener,
                                Constants.EMOTICON_CLICK_TEXT)))
                .setShowDelBtn(EmoticonPageEntity.DelBtnStatus.LAST)
                .setIconUri(ImageBase.Scheme.ASSETS.toUri("xhsemoji_19.png"))
                .build();

**Comment by w446108264** [comment]
the last release version
compile 'com.github.w446108264:XhsEmoticonsKeyboard:2.0.3'


simple

https://github.com/w446108264/XhsEmoticonsKeyboard/blob/master/Simple2/app/src/main/java/com/simple2/MainActivity.java

https://github.com/w446108264/XhsEmoticonsKeyboard/blob/master/XhsEmoticonsKeyboard/app/src/main/java/com/xhsemoticonskeyboard/common/SimpleCommonUtils.java

@kadam5mohit

**Comment by kadam5mohit** [comment]
Thanks for quick reply. last time i have tried to build gradel  compile 'com.github.w446108264:XhsEmoticonsKeyboard:2.0.3'. but its caused some error . let me try again..

**Comment by kadam5mohit** [comment]
i have tried it and it works but chinese characters not coming ..

**Comment by w446108264** [comment]
can you give me a screenshot?

**Comment by w446108264** [comment]
@kadam5mohit

**Comment by kadam5mohit** [comment]
I have just integrate MainActivity code in my app not SimpleCommonUtil.java
![screenshot_2016-06-02-15-38-15](https://cloud.githubusercontent.com/assets/8645535/15741695/78bd7fde-28d8-11e6-8470-668438eb069f.png)

**Comment by w446108264** [comment]
i want to see the screenshot about the problem

i have tried it and it works but chinese characters not coming ..

**Comment by kadam5mohit** [comment]
when i am clicking of right side button in given screenshot then nothing is hapeening

**Comment by w446108264** [comment]
yeah~

it just a simple demo ~
simple2 demo apk download:
https://github.com/w446108264/XhsEmoticonsKeyboard/raw/master/output/simple.apk

you can build the other demo. it's very detailed.  https://github.com/w446108264/XhsEmoticonsKeyboard/tree/master/XhsEmoticonsKeyboard

or download https://github.com/w446108264/XhsEmoticonsKeyboard/raw/master/output/simple.apk
